src/type.ts

```diff
@@ -2574,6 +2579,7 @@ export class TypeResolver {
                 (func) =>
                     (str =
                         str +
+                        'method_' +
```

Comment:
@Shanks0224 We fixed this issue in another PR https://github.com/intel/Wasmnizer-ts/pull/59/files#diff-28a5fe490ceac2e568ef26d4663b154deaaffe37ac184152ae6c87817798b656R2580

Could you restore the code here?

Reply:
I have rolled back this change.